ecl/hql/hqlgram.y

```diff
@@ -11703,6 +11705,12 @@ dedupFlag
                             $$.setExpr(row.getClear(), $1);
                         }
     | dataSet
+    | BEST '(' sortListExpr ')'
```

Comment:
See previous question(s), re. whether there are other non sort list expressions that are applicable to BEST()